addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -88,11 +88,34 @@ export class MessageList extends Component {
                 threadViewer: threadView && threadView.threadViewer,
             };
         });
+        /**
+         * This observer will trigger each time the root element is resized and
+         * adjust the scroll position.
+         */
+        this.resizeObserver = new ResizeObserver(elements => {
+            // In this case, this loop seems useless since we are really working
+            // on a single element, but it's required by the ResizeObserver to
+            // avoid error: ResizeObserver loop completed with undelivered
+            // notifications.
+            for (const element of elements) {
+                if (this.threadView && this.threadView.isAtBottom) {
+                    this._adjustScrollForExtraMessagesAtTheEnd();
```

Comment:
This is actually not consistent with the asc/desc supported mode.
Notice how the rest of the code makes use of "start" and "end", instead of "bottom".
The new variable should be named accordingly, and make sure that it also works when the issue happens on desc lists (in our case, on chatter). But I think it will be ok in that case no matter what due to how it works in browser (height is always compensated on resize from the bottom, so top is never an issue).